src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -1,31 +1,30 @@
 /*
  * File: z_en_sth.c
  * Overlay: ovl_En_Sth
- * Description: Guy looking at moon in South Clock Town
+ * Description: Guy looking at moon in South Clock Town,
+ *              And the Cured Swamp spiderhouse man who gives you Mask of Truth
+ *              And the man who wants to buy Ocean Spiderhouse from you with rupees or Wallet
  */
 
 #include "z_en_sth.h"
-#include "objects/object_sth/object_sth.h"
-#include "objects/object_ahg/object_ahg.h"
-#include "objects/object_mask_truth/object_mask_truth.h"
 
 #define FLAGS (ACTOR_FLAG_1 | ACTOR_FLAG_8)
 
 #define THIS ((EnSth*)thisx)
 
 void EnSth_Init(Actor* thisx, PlayState* play);
 void EnSth_Destroy(Actor* thisx, PlayState* play);
+void EnSth_WaitForObject(Actor* thisx, PlayState* play);
+
+void EnSth_PanicIdle(EnSth* this, PlayState* play);
+void EnSth_HandleOceanSpiderhouseConversation(EnSth* this, PlayState* play);
```

Comment:
Not going to call this out everywhere, but `OceanSpiderhouse` should probably be `OceansideSpiderHouse`, and `SwampSpiderhouse` should be `SwampSpiderHouse`. I think just a few find-and-replaces should take care of it.